__tests__/bundle/bundle-lint-format/.redocly.yaml

```diff
@@ -1,2 +1,3 @@
-apiDefinitions:
-  main: ./openapi.yaml
+apis:
+  main:
```

Comment:
None of the test examples use the `name@version` approach.

Reply:
Anyway, good point. We'll need to write the tests after implementing this functionality.

Reply:
This will be needed for push.